plugins/@grouparoo/customerio/public/templates/app/{{{id}}}.js.template

```diff
@@ -7,7 +7,8 @@ exports.default = async function buildConfig() {
       type: "{{{__pluginName}}}",
       options: {
         siteId: "...", // Customer.io Site ID
-        apiKey: "..." // Customer.io Track API Key
+        apiKey: "...", // Customer.io Tracking API Key
```

Comment:
Non-blocking comment, but to match `appApiKey` and be clearer, now is probably a good time to rename this to `trackApiKey` or `trackingApiKey`